darwin/torch/utils.py

```diff
@@ -16,13 +16,19 @@ def convert_segmentation_to_mask(segmentations: List[Segment], height: int, widt
     """
     Converts a polygon represented as a sequence of coordinates into a mask.
 
-    Input:
-        segmentations: list of float values -> [x1, y1, x2, y2, ..., xn, yn]
-        height: image's height
-        width: image's width
-
-    Output:
-        torch.tensor
+    Parameters
+    ----------
+    segmentations : List[Segment]
+        List of float values -> ``[x1, y1, x2, y2, ..., xn, yn]``.
```

Comment:
If it's a `List[Segment]]` should an example instead be: `[[x1,1 y11, x12, y12], ..., [xn1, yn1, xn2, yn2]]` ?

Reply:
fixed!